src/components/Instructions.tsx

```diff
@@ -20,67 +28,136 @@ type Step = {
 
 const instructionSteps: { [key: number]: Step } = {
   1: {
-    title: "Start the game",
+    title: "Create New Course",
     description: (
       <div>
         {" "}
-        Press <b>Play</b> and fill in your course details.
-        <br /> This should be the course that you would like to evaluate! 📜
+        Press <b>Create New Course</b> if you have not added any courses yet.
+        <br />
       </div>
     ),
     image: stepOneImg,
   },
   2: {
-    title: "Pick the card",
+    title: "Fill in the details",
     description: (
       <div>
-        You will be presented with the first pair of cards. <br /> If the
+        {/* You will be presented with the first pair of cards. <br /> If the
```

Comment:
any reason why this commented instruction is here?

Reply:
Will remove, forgot about those